vital/types/object_track_set.h

```diff
@@ -129,8 +117,40 @@ class VITAL_EXPORT object_track_state : public track_state
     return time_;
   }
 
-  detected_object_sptr detection;
+  void set_detection( detected_object_sptr const& d )
+  {
+    detection_ = d;
+  }
 
+  detected_object_sptr detection()
+  {
+    return detection_;
+  }
+
+  detected_object_scptr detection() const
+  {
+    return detection_;
+  }
+
+  void set_image_point( point_2d const& p )
+  {
+    image_point_ = p;
+  }
+
+  point_2d image_point() const
+  {
+    return image_point_;
+  }
+
+  void set_track_point(point_3d const& p )
```

Comment:
[[ws.space]](../tree/master/doc/code-style.rst) again... more noticeable because you have space on the right but are missing it on the left.